packages/paste-core/components/data-grid/__tests__/index.spec.tsx

```diff
@@ -227,9 +260,13 @@ describe('Data Grid', () => {
       const headerSort = getByTestId('header-sort');
 
       expect(header.getAttribute('aria-sort')).toBe('ascending');
-      headerSort.click();
+      act(() => {
+        headerSort.click();
+      });
       expect(header.getAttribute('aria-sort')).toBe('descending');
-      headerSort.click();
+      act(() => {
+        headerSort.click();
+      });
```

Comment:
praise: Nice. Wonder if we're seeing some speed improvements on these tests now?